doc/develop/extending.rst

```diff
@@ -60,15 +60,18 @@ Start a terminal (with Ctrl+Alt+T) and type::
     sudo apt-get update
     sudo apt-get dist-upgrade
     sudo apt-get install git imagej vim hdfview meld libtiff-tools gimp \
-     python-pyro4-delmic odemis fluodb python-wxtools \
-     python-setuptools python-sphinx inkscape dia-gnome texlive pngcrush cython
+     python3-pyro4-delmic odemis fluodb \
```

Comment:
I suppose you checked if pyro4-delmic is also packaged for python3.
But since this seems like to most likely (and troublesome) part in this change to fail I added a comment as a reminder.

Reply:
Yes. It is packaged on our PPA, and used by Odemis for already more than a year.